compass_app/app/lib/ui/activities/widgets/activities_header.dart

```diff
@@ -0,0 +1,37 @@
+import 'package:flutter/material.dart';
+import 'package:go_router/go_router.dart';
+
+import '../../core/ui/back_button.dart';
+import '../../core/ui/home_button.dart';
+
+class ActivitiesHeader extends StatelessWidget {
+  const ActivitiesHeader({super.key});
+
+  @override
+  Widget build(BuildContext context) {
+    return Padding(
+      padding: const EdgeInsets.only(
+        top: 24,
```

Comment:
Should we should be using constants or something to enforce uniform spacing and padding across widgets?

Reply:
I refactored dimensions in the app a bit, see the latest commits!